cmd/samples/pso/main.go

```diff
@@ -0,0 +1,146 @@
+package main
+
+import (
+	"bytes"
+	"encoding/gob"
+	"flag"
+	"fmt"
+	"reflect"
+	"time"
+
+	"github.com/pborman/uuid"
+	"github.com/uber/cadence-samples/cmd/samples/common"
+	"go.uber.org/cadence/client"
+	"go.uber.org/cadence/encoded"
+	"go.uber.org/cadence/worker"
+)
+
+// gobDataConverter implements encoded.DataConverter using gob for Swarm and Particle
+// WARGNING: Make sure all struct members are public (Capital letter) otherwise serialization does not work!
+// TODO: consider storing blobs in external DB or S3
+type gobDataConverter struct {
```

Comment:
I think it is worth moving into its own file

Reply:
Good idea